cmd/utils.go

```diff
@@ -0,0 +1,32 @@
+package cmd
+
+import (
+	"github.com/fatih/color"
+	printPkg "github.com/kong/deck/print"
+	"github.com/kong/deck/solver"
+	"github.com/spf13/pflag"
+)
+
+func printStats(stats solver.Stats) {
+	// do not use github.com/kong/deck/print because that package
+	// is used only for events logs
+	printFn := color.New(color.FgGreen, color.Bold).PrintfFunc()
+	printFn("Summary:\n")
+	printFn("  Created: %v\n", stats.CreateOps)
+	printFn("  Updated: %v\n", stats.UpdateOps)
+	printFn("  Deleted: %v\n", stats.DeleteOps)
+}
+
+var (
+	silenceEvents bool
+)
+
+func preRunSilenceEventsFlag() error {
+	printPkg.DisableOutput = true
+	return nil
+}
+
+func addSilenceEventsFlag(set *pflag.FlagSet) {
+	set.BoolVar(&silenceEvents, "disable printing events to stdout",
+		false, "silent events")
```

Comment:
Is the arg order here correct? It looks like you have two usage strings and no name: https://pkg.go.dev/github.com/spf13/pflag#FlagSet.BoolVar

`silence-events` seems fine if this is only a partial silence. If it's actually everything, then `silence`.

Reply:
Thanks for catching that, the last refactor gone wrong. Updated.